libtcmu.c

```diff
@@ -390,6 +390,11 @@ void tcmu_block_device(struct tcmu_device *dev)
 {
 	int rc;
 
+	if (!tcmu_is_ring_reset_support(dev)) {
+		tcmu_dev_warn(dev, "reset ring is not supported!\n");
+		return;
+	}
+
```

Comment:
Without this patch, don't we just log a message already?

Reply:
Yes, it was. But it would log a message as an error which will be confusing.

Reply:
Do you mean the "Could not open configfs to write attribute" message from tcmu_set_cfgfs_str was a logged as an error?

What is confusing that one message is a error level but the messages like "Kernel does not support the block_dev action" were warning level? If so, ok.

Reply:
Yes.